Makefile

```diff
@@ -5,15 +5,18 @@ build:
 test:
 	go test -v -race ./... -timeout 40m
 
-e2e-test:
+# traces-dataset.sz is used by ./pkg/tests/end_to_end_tests/ingest_trace_test.go
+pkg/tests/testdata/traces-dataset.sz:
+	curl -L -o ./pkg/tests/testdata/traces-dataset.sz https://github.com/timescale/promscale-test-data/raw/main/traces-dataset.sz
```

Comment:
We might want to check if `curl` exists. eg:
```
ifeq (, $(shell which curl))
 $(error "curl not found. Please install curl")
 endif
```

Reply:
Or we can use `wget` which is present on every distro :)

Reply:
`wget` is not on my mac ;)

Reply:
but yes, I'll switch to `wget`